source/getting-started/javascript.md

```diff
@@ -90,48 +95,289 @@ See also: [app authorization in the API reference](/reference/#authorizing-your-
 var connection = new pryv.Connection(credentials);
 ```
 
+### Initialize the datastore
```

Comment:
rename to "Fetch the stream structure". "Datastore" is an internal mechanism that doesn't need to be mentionned.